framework/src/modules/poa/internal_method.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { BaseMethod } from '../base_method';
+
+export class PoAMethod extends BaseMethod {}
```

Comment:
missed out, but it should be `BaseInternalMethod`
```suggestion
export class PoAMethod extends BaseInternalMethod {}
```